network-api/networkapi/wagtailpages/templates/wagtailpages/blocks/article_teaser_block.html

```diff
@@ -0,0 +1,42 @@
+{% extends "./base_streamfield_block.html" %}
+{% load wagtailcore_tags wagtailimages_tags i18n localization %}
+
+{% block block_content %}
+  <div class="tw-row tw-py-4">
+	{% for block in self.cards %}
+      {% with card=block %}
+        {% comment %}
+        Check for the card.article, sometimes and article can get deleted
+        causing dangling references in streamfield blocks and passing None to
+        the `pageurl` template tag would throw an error.
+        {% endcomment %}
+        {% if card.article %}
+          <div class="tw-px-4 tw-w-full medium:tw-w-1/3 tw-mb-5">
+            <div class="tw-bg-white tw-relative tw-py-4 tw-w-full tw-flex tw-flex-col">
+              <a href="{% pageurl card.article %}">
+                <span class="tw-h5-heading tw-mb-2">{{ card.article.title }}</span>
+              </a>
+            </div>
+
+            <div class="blog-authors d-flex flex-wrap align-items-center">
+              <div class="tw-mx-2">
+                {% for author in card.article.authors.all %}
+                  {% with author_image_classes="-tw-ml-[0.7rem] tw-overflow-hidden tw-rounded-full tw-border-2 tw-border-white tw-inline-block tw-w-[40px] tw-h-[40px] tw-border-r d-flex align-items-center justify-content-center last:tw-mr-2" no_image_classes="tw-bg-[url('../_images/blog-author-placeholder.jpg')] tw-bg-cover "%}
+                    <div class="author-image {{ author_image_classes }} {% if author.author.image is None %} {{no_image_classes}} {% endif %}" aria-label="{{ author.author.name }}">
+                      {% if author.author.image %}
+                        <img src="{% image_url author.author.image "fill-100x100" %}" alt="{{ author.author.name }}" title="{{ author.author.name }}" />
+                      {% endif %}
+                    </div>
+                  {% endwith %}
```

Comment:
Argh... too bad [the fragment](https://github.com/mozilla/foundation.mozilla.org/blob/69568d67a91f08bb180eb3adfdf4fff38942d599/network-api/networkapi/templates/fragments/profile_images_stack.html) is not on `main` yet 🤦‍♂️  Not your fault of course.